module6/src/main/java/bean/DirectoriesScanning.java

```diff
@@ -0,0 +1,57 @@
+package bean;
+
+import writer.WriterWrapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class DirectoriesScanning {
+    public final static String DIR_PREFIX = "|----";
+    public final static String FILE_PREFIX = "|   ";
+    private final static String DEFAULT_INDENTATION = "   ";
+
+    /**
+     *This method scanning directory and writing results to file.
+     *
+     * @param depth difference between directories levels.
+     * @param file path to directory
+     * @param writer WriterWrapper object
+     * @throws IOException
+     */
+    public static void scanningFilesInDirectory(int depth, File file, WriterWrapper writer) throws IOException {
+        StringBuilder indentation = new StringBuilder();
+        File[] files;
+        for (int i = 0; i < depth; i++) {
+            indentation.append(DEFAULT_INDENTATION);
+        }
+        if (file.isDirectory()) {
+            writer.writeLine(indentation + DIR_PREFIX + file.getName() + "\n");
+        } else {
+            writer.writeLine(indentation + FILE_PREFIX + file.getName() + "\n");
+        }
+        if (file.isDirectory()) {
+            files = file.listFiles();
+            sortFileWrappers(Arrays.asList(files));
+            for (File value : files) {
+                scanningFilesInDirectory(depth + 1, value, writer);
+            }
+        }
+    }
+    private static void sortFileWrappers(List<File> childFiles) {
```

Comment:
Missing empty line.
Please, use default IDEA formatter to avoid such comments further.